src/uct/api/uct.h

```diff
@@ -217,7 +217,8 @@ typedef struct uct_md_resource_desc {
 enum uct_component_attr_field {
     UCT_COMPONENT_ATTR_FIELD_NAME              = UCS_BIT(0), /**< Component name */
     UCT_COMPONENT_ATTR_FIELD_MD_RESOURCE_COUNT = UCS_BIT(1), /**< MD resource count */
-    UCT_COMPONENT_ATTR_FIELD_MD_RESOURCES      = UCS_BIT(2)  /**< MD resources array */
+    UCT_COMPONENT_ATTR_FIELD_MD_RESOURCES      = UCS_BIT(2), /**< MD resources array */
+    UCT_COMPONENT_ATTR_FIELD_CAP_FLAGS         = UCS_BIT(3)  /**< Capability flags */
```

Comment:
doesn't FLAGS always imply some sort of capability? IMHO it should be UCT_COMPONENT_ATTR_FIELD_FLAGS. (same for UCT_COMPONENT_CAP_FLAG_CM)